code/game/jobs/job/church.dm

```diff
@@ -1,8 +1,9 @@
 /datum/job/chaplain
 	title = "NeoTheology Preacher"
 	flag = CHAPLAIN
+	head_position = 1
 	department = DEPARTMENT_CHURCH
-	department_flag = CHURCH
+	department_flag = CHURCH | COMMAND
```

Comment:
@GrayRachnid yup I already tested to make sure the preacher appears under command on the manifest, this is the line that causes that.